SampleProjects/TestSomething/test/wire.cpp

```diff
@@ -0,0 +1,68 @@
+#include <ArduinoUnitTests.h>
+#include <Arduino.h>
+#include <Wire.h>
+using namespace std;
```

Comment:
Let's remove this line (`using namespace std;`) and add `std::` to the front of each reference to `deque`. (This isn't very important but I'd like an edit to add a newline to the end of the file and sometimes git doesn't recognize that as enough of a change to commit/push).